scss/_patterns_equal-height-row.scss

```diff
@@ -0,0 +1,147 @@
+@import 'settings';
+
+@mixin vf-p-equal-height-row {
+  .p-equal-height-row {
+    display: grid;
+    grid-template-columns: subgrid;
+    position: relative;
+
+    .p-equal-height-row__col {
+      border-top-color: $colors--theme--border-default;
+      border-top-style: solid;
+      border-top-width: 1px;
+      display: grid;
+      grid-column: span $grid-columns-small;
+      grid-row: span 4;
+      grid-template-rows: subgrid;
+      position: relative;
+
+      .p-equal-height-row__col-item {
+        grid-column: span $grid-columns-small;
+      }
+
+      .p-equal-height-row__col-item:first-child {
+        grid-row: auto;
+      }
+
+      @media screen and ($breakpoint-small <= width < $breakpoint-large) {
+        grid-column: span $grid-columns-medium;
+        grid-template-columns: subgrid;
+        margin-top: -1px;
+
+        .p-equal-height-row__col-item {
+          grid-column: span calc($grid-columns-medium / 2);
+        }
+
+        .p-equal-height-row__col-item:first-child {
+          grid-row: span 100;
+        }
+      }
+
+      @media screen and (width >= $breakpoint-large) {
+        border: none;
+        grid-column: span calc($grid-columns / 4);
+      }
+    }
+
+    // divider styles
+    // remove row level dividers for small screen
+    &[class*='p-equal-height-row--divider-below-item-']::before,
+    &[class*='p-equal-height-row--divider-below-item-']::after {
+      @extend %vf-pseudo-border;
+      display: none;
+      width: 100%;
+    }
+
+    // For smaller screens, the divider pseudo elements are inserted at the column level
+    &.p-equal-height-row--divider-below-item-1 .p-equal-height-row__col::before,
```

Comment:
Would it make sense to use actual border on small screens instead of the before/after pseudo element divider?

So, if we have a class name that adds divider between items 1/2 - the item 2 will have border on top:

```scss
// pseudo-code, just to give idea
&.has-divider-above-item-2 .p-equal-height-row__item:nth-child(2) {
  border-top: 1px solid $colors--theme--border-default;
}
```